rest/route/stats.go

```diff
@@ -344,3 +374,97 @@ func (tsh *testStatsHandler) makeStartAtKey(testStats model.APITestStats) string
 func makeGetProjectTestStats(sc data.Connector) gimlet.RouteHandler {
 	return &testStatsHandler{sc: sc}
 }
+
+///////////////////////////////////////////////
+// /projects/<project_id>/task_stats handler //
+///////////////////////////////////////////////
+
+type taskStatsHandler struct {
+	sc data.Connector
+	statsHandler
+}
+
+func (tsh *taskStatsHandler) Factory() gimlet.RouteHandler {
+	return &taskStatsHandler{sc: tsh.sc}
+}
+
+func (tsh *taskStatsHandler) Parse(ctx context.Context, r *http.Request) error {
+	vals := r.URL.Query()
+	project := gimlet.GetVars(r)["project_id"]
```

Comment:
I'd avoid declaring these values here and just put these calls on lines 394 and 396 respectively.

Reply:
Done.